.github/workflows/cmake-build-example-win.yml

```
@@ -0,0 +1,48 @@
+      matrix:
+        build_type: [Release, Debug]
+    steps:
+    - name: Cache Conan
+      uses: actions/cache@v3
+      with:
+        path: ~/.conan/data
+        key: ${{ runner.os }}-conan-${{ hashFiles('conan.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-conan-
+
+    - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)